src/bin/console/console.cpp

```diff
@@ -896,6 +901,20 @@ int Console::_change_runtime_setting(const std::string& input) {
     return 0;
   }
 
+  if (property == "binary_caching") {
+    if (value == "on") {
+      _binary_caching = true;
+      out("Binary caching turned on\n");
```

Comment:
Trailing period?

Reply:
Well, it's not a full sentence. Hmm ..

Reply:
So, what's your take? We are kinda inconsequent in the console.

Reply:
Without periods as those are not full sentences?